### PR TITLE
Add commands useful for presentation

### PR DESCRIPTION
**Pull request by JanAnders**
This PR adds a few commands to try out the application.

`make training` trains the model
`make inference` starts the inference locally
`make tensorboard` starts tensorboard
`make inf_request` and `make inf_request2` send different requests to the deployed inference server. Add the public inference server IP at the top of the file first!

In addition, clean up the workflow files so they only run on push to main.

### CODE REVIEW

File: .github/workflows/build_training.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   build:
+
```
Comment (mkoehnenlmu) [inline .github/workflows/build_training.yml:12]: interesting change :D

File: .github/workflows/run_monitoring.yml [hunk 2]
```diff
@@ -16,7 +14,7 @@ on:
 # https://github.com/google-github-actions/auth#setup
 
 jobs:
-  inference:
```
Comment (mkoehnenlmu) [inline .github/workflows/run_monitoring.yml:19]: was this causing the problems?

**Review by mkoehnenlmu — APPROVED** [review]
Great great great additions. Now erverything really works with a snap :D

File: .github/workflows/run_monitoring.yml (hunk 2 above)
Comment (JanAnders) [reply .github/workflows/run_monitoring.yml:19]: Nope, just the wrong name in the UI (Which can be confusing) :)

File: .github/workflows/build_training.yml (hunk 1 above)
Comment (JanAnders) [reply .github/workflows/build_training.yml:12]: I wanted all three files to have the same structure :P

**Pull request merged**